.github/workflows/maven.yml

```diff
@@ -29,3 +29,14 @@ jobs:
         cache: maven
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+
+  checkstyle:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Is there any way to have this run as a pre commit git hook?

Reply:
added pre-commit to run checkstyle